packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -209,20 +209,19 @@ export const prepareTransactionFeedKit = (zone, zcf) => {
           let lastEvidence;
           for (const store of found) {
             const next = store.get(txHash);
-            if (lastEvidence) {
-              if (keyEQ(lastEvidence, next)) {
-                lastEvidence = next;
-              } else {
-                trace(
-                  '🚨 conflicting evidence for',
-                  txHash,
-                  ':',
-                  lastEvidence,
-                  '!=',
-                  next,
-                );
-                Fail`conflicting evidence for ${quote(txHash)}`;
-              }
+            if (lastEvidence && !keyEQ(lastEvidence, next)) {
+              // Ignore conflicting evidence, but treat it as an error
+              // because it should never happen and needs to be prevented
+              // from happening again.
+              trace(
+                '🚨 conflicting evidence for',
+                txHash,
+                ':',
+                lastEvidence,
+                '!=',
+                next,
```

Comment:
do we have the operator ids handy at this point?
It would be nice to know which evidence comes from which operator.
I think it's not hard to find from a few earlier log messages, though.